src/Neo.CLI/CLI/MainService.cs

```diff
@@ -521,13 +521,13 @@ private void WriteBlocks(uint start, uint count, string path, bool writeStart)
                 if (writeStart)
                 {
                     fs.Seek(sizeof(uint), SeekOrigin.Begin);
-                    fs.Read(buffer, 0, buffer.Length);
+                    fs.ReadExactly(buffer);
                     start += BitConverter.ToUInt32(buffer, 0);
                     fs.Seek(sizeof(uint), SeekOrigin.Begin);
                 }
                 else
                 {
-                    fs.Read(buffer, 0, buffer.Length);
+                    fs.ReadExactly(buffer);
```

Comment:
this is the only place where code is changed in the core. @shargon

Reply:
This paragraph was automatically changed after running `dotnet format` in dotnet 9, and I saw that the logic hadn't changed, so I submitted it.

Reply:
turn it off in the `.editorconfig`